app/institutions/dashboard/-components/object-list/contributors-field/component.ts

```diff
@@ -48,17 +50,31 @@ export default class InstitutionalObjectListContributorsField extends Component<
             const contributor = getContributorById(contributors, getSingleOsfmapValue(attribution, ['agent']));
             const roleIri: AttributionRoleIris = getSingleOsfmapValue(attribution, ['hadRole']);
             return {
-                name: getSingleOsfmapValue(contributor,['name']),
+                name: getSingleOsfmapValue(contributor, ['name']) || this.intl.t('general.unknownContributor'),
+                userId: contributor['@id'],
+                nodeId: searchResult.indexCard.get('osfGuid'),
                 url: getSingleOsfmapValue(contributor, ['identifier']),
                 permissionLevel: this.intl.t(roleIriToTranslationKey[roleIri]),
             };
         });
+
+    }
+
+    @action
+    handleOpenModal(contributor: any) {
+        this.args.projectRequestModal(contributor);
+    }
+
+    get shouldShowButton() {
+        const { searchResult } = this.args;
+        const { resourceMetadata } = searchResult;
+        return resourceMetadata.resourceType[0]['@id'] === 'Project';
     }
 }
 
 function hasInstitutionAffiliation(contributors: any[], attribution: any, institutionIris: string[]) {
     const attributedContributor = getContributorById(contributors, getSingleOsfmapValue(attribution, ['agent']));
-
+    return true;
```

Comment:
Is this supposed to be there? Looks like it skips the rest of the function.

Reply:
Good catch, this was for debugging, it should have been visible on staging3 though, maybe they missed it? Anyway I will remove it.